### PR TITLE
feat(edit-contentlet): Implement new design in the edit content screen sidebar #27908

### LINKED ISSUE #27908: Edit Content: Implement new design in the edit content screen sidebar
### Parent
#25445 

### User Story
As a content editor, I want the sidebar in the edit content screen to reflect the new design provided, so that the interface is consistent and visually appealing.

https://github.com/dotCMS/core/assets/751424/4d3c33da-9cf9-45e2-8375-ad4d31a454a4

### Acceptance Criteria
- The sidebar design matches the provided Angular components, HTML, and CSS.
- All functionality in the sidebar remains intact after the design implementation.
- The new design is responsive and works well on different screen sizes.

### Proposed Objective
Core Features

### Proposed Priority
Priority 3 - Average

### External Links
- [Figma](https://www.figma.com/file/YMTdSO5gKf18rc1pCCv1jN/Edit-Contentlet?type=design&node-id=4308%3A107601&mode=design&t=uVy7zKGF9OK0dONt-1)
- [Prototype](https://www.figma.com/proto/YMTdSO5gKf18rc1pCCv1jN/Edit-Contentlet?page-id=365%3A9805&type=design&node-id=4308-107601&viewport=-16104%2C-11033%2C0.5&t=mHqvvKdDVgGZPhlh-1&scaling=min-zoom&starting-point-node-id=4308%3A107601&mode=design)

### Assumptions & Initiation Needs
There are no dependencies or assumptions for this feature implementation.

### Quality Assurance Notes & Workarounds
N/A

### Sub-Tasks & Estimates
N/A

### PR DESCRIPTION
**Pull request by rjvelazco**
* Implement the new design in the edit content screen sidebar

### Checklist
- [x] Tests
- [x] Translations
- [x] Security Implications Contemplated (add notes if applicable)

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
issue-27908-edit-content-implement-new-design-in-the-edit-content-screen-sidebar  | fix-issue-27908-edit-content-implement-new-design-in-the-edit-content-screen-sidebar

### CODE REVIEW

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        line-height: 140%;
+    }
+
+    .content-history__author {
+        overflow: hidden;
+        color: $color-palette-gray-800;
+        text-overflow: ellipsis;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 400;
```
Comment (zJaaal) [inline core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:129]: Variables

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+.item__description {
+    color: $color-palette-gray-800;
+    margin: 0;
+}
+
+.workflow-column__title,
+.workflow-column__description {
+    color: $black;
+    height: 32px;
+    font-size: 14px;
```
Comment (zJaaal) [inline core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss:31]: variable

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    line-height: $line-height;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```
Comment (zJaaal) [inline core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss:35]: Check if the mixin I mentioned before works here too

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.scss [hunk 4]
```diff
@@ -1,34 +1,64 @@
...
-    p {
-        color: $color-palette-gray-800;
-        margin: 0;
-    }
+            label {
+                color: $color-palette-primary-500;
+                font-size: 12px;
+                font-style: normal;
+                font-weight: 400;
+                line-height: 140%;
```
Comment (zJaaal) [inline core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.scss:46]: variables

**Review by zJaaal — CHANGES_REQUESTED** [review]

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+        line-height: 140%;
+    }
+}
+
+.content-history__container {
+    display: flex;
+    align-items: center;
+    flex: 1 0 0;
+    align-self: stretch;
+    border-radius: 8px;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:73]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+        i {
+            color: $color-palette-primary-500;
+        }
+    }
+
+    .sidebar-card__label {
+        color: $color-palette-gray-800;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 400;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:63]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+        }
+    }
+
+    .sidebar-card__header {
+        display: flex;
+        justify-content: space-between;
+        color: $black;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 600;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:51]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+        &:hover {
+            background: $color-palette-primary-100;
+        }
+    }
+
+    .sidebar-card__header {
+        display: flex;
+        justify-content: space-between;
+        color: $black;
+        font-size: 14px;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:49]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+            top: 0;
+            bottom: 0;
+            margin: auto;
+        }
+    }
+
+    .content-history__title {
+        color: $black;
+        font-size: 14px;
+        font-weight: 600;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:119]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss (hunk 1 above)
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:129]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+        font-style: normal;
+        font-weight: 400;
+        line-height: 140%;
+    }
+
+    .content-history__date {
+        color: $color-palette-gray-700;
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 400;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:137]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss (hunk 2 above)
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss:31]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.scss (hunk 4 above)
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.scss:46]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+        height: 100%;
+        padding: $spacing-1;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: $spacing-0;
+        flex: 1 0 0;
+        position: relative;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-information/dot-content-aside-information.component.scss:87]: Done!

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss (hunk 3 above)
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss:35]: Done!

**Review by zJaaal — APPROVED** [review]

File: core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.ts [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+    providers: [DotWorkflowService],
+    templateUrl: './dot-content-aside-workflow.component.html',
+    styleUrl: './dot-content-aside-workflow.component.scss'
+})
+export class DotContentAsideWorkflowComponent implements OnInit, OnChanges {
+    @Input() inode: string;
+    @Input() contentType: DotCMSContentType;
+
+    private readonly workflowService = inject(DotWorkflowService);
+    protected readonly workflowStatus = signal<DotCMSWorkflowStatus>(null);
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.ts]: Done!

**Review by KevinDavilaDotCMS — APPROVED** [review]

**Pull request merged**